db/schema.rb

```diff
@@ -10,9 +10,9 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
```

Comment:
@jerry-74 These changes are not related to this Pull request, So its suggested to revert back to the original file.